notes/seed.py

```diff
@@ -16,7 +16,7 @@
     raise Exception('Zip file path is not provided')
 zip_file_path = sys.argv[1]
 
-folder_to_extract_files_from = 'Takeout/Keep'
+folder_to_extract_files_from = 'Keep'
 folder_with_extracted_files = os.path.join(os.getcwd())
 
```

Comment:
let's add code to delete temporary folder extracted from zip archive at the end of this file

```suggestion
from pathlib import Path
extracted_temporary_folder_path = Path(settings.NOTES_SOURCE_DIR).resolve().parent
shutil.rmtree(extracted_temporary_folder_path)
```